src/airflow/airqo_etl_utils/config.py

```diff
@@ -170,7 +170,7 @@ class Config:
     MLFLOW_TRACKING_URI = os.getenv("MLFLOW_TRACKING_URI")
     FORECAST_MODELS_BUCKET = os.getenv("FORECAST_MODELS_BUCKET")
     MONGO_URI = os.getenv("MONGO_URI")
-    MONGO_DATABASE_NAME = os.getenv("MONGO_DATABASE_NAME")
+    MONGO_DATABASE_NAME = os.getenv("MONGO_DATABASE_NAME", "airqo_db")
```

Comment:
Just a reminder to add this to the .env after this has been merged
@Mnoble-19 
@Baalmart

Reply:
Thanks @BenjaminSsempala , let me actually just remove it, and use an alternative approach